lib/parser.js

```diff
@@ -68,10 +77,48 @@ module.exports = class Parser {
         dockerFileName = build.dockerfile
       }
     }
+    if (Parser.isGitHubUrl(buildPath)) {
+      return path.resolve('/', dockerFileName)
+    }
     return path.resolve('/', buildPath, dockerFileName)
   }
 
   /**
+   * Parser for `code`
+   *
+   * @param {Object}          opts
+   * @param {String|Object}   opts.build
+   * @param {Object}          opts.warnings
+   * @return {Object|Null}    code object with `repo` and `commitish` which can be null.
+   */
+  static buildRemoveCodeParser (opts) {
+    const build = opts.build
+    const warnings = opts.warnings
+    if (!build || build.disabled) return null
+
+    if (build.args) {
+      warnings.add('The `args` argument is not supported for builds. These args will not be passed to the build.', { args: build.args })
+    }
+
+    let buildPath
+    if (typeof build === 'string') {
+      buildPath = build
+    } else {
+      buildPath = build.context
+    }
+    if (Parser.isGitHubUrl(buildPath)) {
```

Comment:
Nit: I think this logic should be the other way around. `if (!Parser.isGitHubUrl(buildPath)) return`. Removes unnecessary indentation and makes the code easier to read.